### DISCUSSION TITLE
guess what i did on reddit?

### DISCUSSION THREAD

**Discussion by Outfit8TSB** [GAME-CLI-SRV-DEV/GCSD-Discussions#46] [Announcements]
i asked @Zekiah-A to bring me 32 palette back.

**Comment by Outfit8TSB** [comment]
if they bring it back i am definitely drawing my plugin logo

**Reply by Outfit8TSB** [reply]
guys,
are ViaVersion and ProtocolSupport had a Subreddit called r/viaversion and r/protocolsupport?

**Reply by Outfit8TSB** [reply]
i guess geyser does so i draw geyser if @Zekiah-A bring 32 palette back to meh

**Reply by Outfit8TSB** [reply]
hope viaproxy.geysermc.org:25568 exist
so i can draw address_port_version.viaproxy.geysermc.org:25568/19132

**Reply by Outfit8TSB** [reply]
well i will draw lenni0451's viaproxy srv instance and my viaproxy srv instance address format.

**Comment by Outfit8TSB** [comment]
here is the post https://www.reddit.com/r/placetk/comments/1ggd5x7/zek_bring_this_back/

**Reply by Zekiah-A** [reply]
Yes, I might consider this

**Reply by Outfit8TSB** [reply]
thanks dude

**Comment by Outfit8TSB** [comment]
also here is the mapping of darkplace colour
![image](https://github.com/user-attachments/assets/cc64d2e3-3546-4aad-92b3-a74a3fbf5cf3)

**Reply by Outfit8TSB** [reply]
@Zekiah-A i will return to rplace.live soon when the 32 color is back
while then i am just going to map rplace.live on pixelplayground
so dont worry about me
